CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+## 14.1.0 (10/16/23)
+
+### New features
+
+Teleport Connect 14.1 introduces Connect My Computer which makes it possible to add your personal machine to a Teleport cluster in just a couple of clicks. Whether you're exploring capabilities Teleport or want to make your computer available in your private cluster, Connect My Computer lets you do that without having to use the terminal to get the job done. Docs: https://goteleport.com/docs/connect-your-client/teleport-connect/#connect-my-computer
```

Comment:
If you're mentioning CMC, then I feel like we should mention the other major things that were outlined on our upcoming releases page as well like Access Reviews, Access Monitoring and Agentless EC2.